libfwupd/fwupd-enums.h

```diff
@@ -91,6 +91,7 @@ typedef enum {
  * @FWUPD_DEVICE_FLAG_HISTORICAL		Device is for historical data only
  * @FWUPD_DEVICE_FLAG_ENSURE_SEMVER:		Ensure the version is a valid semantic version, e.g. numbers separated with dots
  * @FWUPD_DEVICE_FLAG_ONLY_SUPPORTED:		Only devices supported in the metadata will be opened
+ * @FWUPD_DEVICE_FLAG_WILL_DISSAPPEAR:		Device will disappear after update and can't be verified
```

Comment:
One 'S', no?

Reply:
yes, this is one of those words I always mess up and have auto-correct usually fix for me.  will fixup.